### PR TITLE
VB-2373 SQL generator updates for WEEKLY_FREQUENCY and ACTIVE flag.

### PR DESCRIPTION
**Pull request by dhirajshettymoj**
## What does this pull request do?

SQL generator updates for WEEKLY_FREQUENCY and ACTIVE flag.

## What is the intent behind these changes?

Updates based on column updates and new column additions to session_template table.

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/utils/SessionTemplateSQLGeneratorApplication.kt [hunk 1]
```diff
@@ -276,9 +278,10 @@ class SessionTemplateSQLGenerator {
       endDate = sessionRecord.get(END_DATE.name)?.let { LocalDate.parse(it) },
       dayOfWeek = DayOfWeek.valueOf(sessionRecord.get(DAY_OF_WEEK.name).uppercase()),
       locationKeys = sessionRecord.get(LOCATION_KEYS.name)?.uppercase(),
-      biWeekly = sessionRecord.get(BI_WEEKLY.name)?.uppercase().toBoolean(),
+      weeklyFrequency = Integer.parseInt(sessionRecord.get(WEEKLY_FREQUENCY.name)),
```
Comment (AledWEvans) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/utils/SessionTemplateSQLGeneratorApplication.kt:281]: you could check that this is greater than zero,..

**Review by AledWEvans — APPROVED** [review]
LGTM one comment your call

**Review by AledWEvans — APPROVED** [review]
LGTM

File: src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/utils/SessionTemplateSQLGeneratorApplication.kt (hunk 1 above)
Comment (dhirajshettymoj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/utils/SessionTemplateSQLGeneratorApplication.kt:290]: added validation

**Pull request merged**